crates/bevy_image/src/image.rs

```diff
@@ -4,6 +4,8 @@ use super::basis::*;
 use super::dds::*;
 #[cfg(feature = "ktx2")]
 use super::ktx2::*;
+#[cfg(not(feature = "bevy_reflect"))]
+use bevy_reflect::TypePath;
```

Comment:
Eventually it might be worth splitting `TypePath` out into its own crate so we don't have to continue pulling in `bevy_reflect` when it's not needed 🤔